ground/gcs/src/plugins/picoc/picocgadgetwidget.cpp

```diff
@@ -385,6 +385,17 @@ void PicoCGadgetWidget::sleep(int msec)
 }
 
 /**
+* @brief PicoCGadgetWidget::on_tbTestValueSend_clicked  Send the Test Value
+ */
+void PicoCGadgetWidget::on_tbTestValueSend_clicked()

```

Comment:
These signal/slots connections based on widget name while easier/faster to code are IMO harder to read and aren't usually used on the GCS code. Can you change it please?